### DISCUSSION TITLE
prepared statement “pdo_stmt_00000005” does not exist

### DISCUSSION THREAD

**Discussion by shoaibsharif** [supabase/supabase#1435] [Questions] [answered]
I recently migrated my database from heroku to Supabase. For some reason, I keep getting error in my database. 

```bash
PDOException: SQLSTATE[26000]: Invalid sql statement name: 7 ERROR:  prepared statement "pdo_stmt_00000005" does not exist
```
I actually wanted to use supabase postgresql database for my Laravel project.

**Comment by silentworks** [comment, marked as answer]
It's hard to tell what could be the cause of this without seeing your DB schema, maybe something didn't copy across from Heroku correctly. Do a DB dump from Heroku, load the data up into a local instance of Postgres on your computer, point your Laravel project to it and see if you get the same error.

It also looks like someone on StackOverflow replied to your post with this link https://stackoverflow.com/questions/60108749/laravel-data-mismatch-error-while-using-pdoattr-emulate-prepares-true#answer-60223442
